routemaster/server/endpoints.py

```diff
@@ -167,6 +167,51 @@ def create_label(state_machine_name, label_name):
         abort(409, msg)
 
 
+@server.route(
+    '/state-machines/<state_machine_name>/labels/<label_name>/restore_and_restart', # noqa
```

Comment:
Please could we pin down this `noqa` with a clarification of specifically which error we're silencing? Just as we would prefer not to blanket catch exceptions, we should prefer not to blanket silence errors.

In case you're not aware, the syntax is like: `noqa: E123`.